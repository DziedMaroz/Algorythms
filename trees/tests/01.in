1
2
3
4
15
20
22
11
10
9
8

