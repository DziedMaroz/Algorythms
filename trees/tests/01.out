1
2
3
4
15
11
10
9
20
22
